src/game/Pet.cpp

```diff
@@ -932,6 +932,9 @@ void Pet::GivePetXP(uint32 xp)
     if (level >= maxlevel)
         return;
 
+    if (float rateXP = sWorld.getConfig(CONFIG_FLOAT_RATE_PET_XP_KILL))
```

Comment:
Does it look good now? I removed the check I had written into the imported commit and instead in my allignment commit I set it properly as positive, in the proper value load checks like you said, as well as setting some of the others for positive there since negative rates don't make sense, right?

So what do you think, you think it looks good or is the allignment itself redundant and a waste of time?